src/rebase.c

```diff
@@ -975,10 +978,45 @@ static int rebase_commit__create(
 		message = git_commit_message(current_commit);
 	}
 
-	if ((error = git_commit_create(&commit_id, rebase->repo, NULL, author,
-		committer, message_encoding, message, tree, 1,
-		(const git_commit **)&parent_commit)) < 0 ||
-		(error = git_commit_lookup(&commit, rebase->repo, &commit_id)) < 0)
+	/* this error will be cleared by the signing process, but should be set
+	 * to signal the unsigned commit create process if we are not going to sign */
```

Comment:
Nit: multiline-comments should start with empty `/*` and `*/` lines. No need to fix this, though

Reply:
When I updated the flow to always use `git_commit_create_with_signature`, this comment no longer mattered.